File: Samples/Beyond.NET.Sample.Swift/Tests/TestClasses/WebProxyTests.swift
import XCTest
import BeyondDotNETSampleKit

final class WebProxyTests: XCTestCase {
    override class func setUp() {
        Self.sharedSetUp()
    }

    override class func tearDown() {
        Self.sharedTearDown()
    }

    func testWebProxy() throws {
        let dnProxy = try Beyond_NET_Sample_WebProxyTests.createWebProxy()
        let dnCredential = try dnProxy.credentials
        XCTAssertNil(dnCredential)
        
        let proxy = try System_Net_WebProxy()
        let credential = try proxy.credentials
        XCTAssertNil(credential)
    }
}

File: Samples/Beyond.NET.Sample.Android/app/src/androidTest/java/com/example/beyondnetsampleandroid/WebProxyTests.kt
package com.example.beyondnetsampleandroid

import androidx.test.ext.junit.runners.AndroidJUnit4

import org.junit.Test
import org.junit.runner.RunWith

import org.junit.Assert.*

import com.example.beyondnetsampleandroid.dn.*

@RunWith(AndroidJUnit4::class)
class WebProxyTests {
    @Test
    fun testWebProxy() {
        val dnProxy = Beyond_NET_Sample_WebProxyTests.createWebProxy()
        val dnCredential = dnProxy.credentials
        assertNull(dnCredential)

        val proxy = System_Net_WebProxy()
        val credential = proxy.credentials
        assertNull(credential)
    }
}
